packages/dashboards/src/lib/components/list-widget/list-widget.component.ts

```diff
@@ -70,10 +70,23 @@ export class ListWidgetComponent
 
     public ngOnInit(): void {
         this.initResizeObserver();
+        this.data = [];
     }
 
     public ngOnChanges(changes: SimpleChanges): void {
+        //this.data = [true,false, 1, 0, "", undefined, null, "ajajaja"]
+        this.data = this.data?.map((x) => {
```

Comment:
This will loop through all values after each change (since it is in ngOnChanges function). If there are many values, won't that slow down performance?